fs/Kconfig

```diff
@@ -129,11 +129,17 @@ config FS_HEAPBUF_SECTION
 
 config FS_REFCOUNT
 	bool "File reference count"
-	default !DEFAULT_SMALL
+	default n
```

Comment:
I think the default should be !DISABLE_PTHREAD

Reply:
in nuttx, is pthread the only way to share files between threads?
how about eg. aio?

IMO, FS_REFCOUNT should not be a user-visible option at all.
we should just enable it automatically when a user enabled functionalities which possibly need it.